### DISCUSSION TITLE
Manually specifying a monitor as non-Apple display?

### DISCUSSION THREAD

**Discussion by qvacua** [MonitorControl/MonitorControl#852] [General]
I have the following setting: MacBook Pro (M1 Pro) with macOS 12.1 + LG UltraFine 4K 21.5" + Dell U2719D. Sometimes–I could not find out when exactly this happens–macOS recognizes the Dell monitor as brightness-controllable monitor, i.e. the Display settings or the Control Center shows a brightness slider for it. When I use the slider, the brightness of the LG monitor changes 😞.

MonitorControl thinks that the Dell monitor is an `model/AppleDisplay` since the system returns a value for the brightness (if I read the code correctly; `DisplayManager.isAppleDisplay()`). And it changes the brightness of the LG monitor when I use the slider for the Dell monitor (as expected).

Would it be an option to include an option to "force" a monitor to be a `model/OtherDisplay`? I've filed a bug at Apple, but I guess this won't be fixed...

**Comment by waydabber** [comment]
Hi @qvacua - I think you should add a Feature Request with a title "Force DDC control when macOS display as Apple display". It could be implemented quite easily.

**Reply by waydabber** [reply]
Or I can convert this discussion into an issue. :)

**Reply by waydabber** [reply]
https://github.com/MonitorControl/MonitorControl/issues/853

**Reply by qvacua** [reply]
Thanks!
